### PR TITLE
Azure plugin

### PR DESCRIPTION
**Pull request by sirferl**
This is the Plugin for uploading certificates to Azure Key-Vaults.

### CODE REVIEW

**Comment by hosseinsh** [comment]
Thanks for adding this Plugin to Lemur!
This would benefit a lot from testing @sirferl, maybe a PR in the near future?

File: lemur/plugins/lemur_azure_dest/plugin.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+from lemur.plugins.bases import DestinationPlugin
+
+from cryptography.hazmat.primitives import serialization
+import requests
+import json
+import sys
+import base64
+
+
+def base64encode(string):
```
Comment (hosseinsh) [inline lemur/plugins/lemur_azure_dest/plugin.py:25]: I wonder if this should move into util

https://github.com/Netflix/lemur/blob/bde3d1ac66a50c91cf8bc6daab001b475115e1d9/lemur/plugins/lemur_kubernetes/plugin.py#L78-L79

Comment (sirferl) [reply lemur/plugins/lemur_azure_dest/plugin.py]: moved this function to common.utils

**Comment by sirferl** [comment]
Hello, sorry, I am very busy at the moment with other stuff. Could you please merge in the changes. I will create a test set later on. Next project will be the entrust-source plugin extension.

**Review by hosseinsh — APPROVED** [review]

**Comment by hosseinsh** [comment]
sure thing @sirferl.
All the best of luck with the ongoing projects, and thanks again for contributing this plugin to Lemur!

**Comment by sirferl** [comment]
@hosseinsh thank you !

**Pull request merged**